test-app/tests/integration/components/bs-collapse-test.ts

```diff
@@ -1,13 +1,26 @@
 import { module } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render, settled, waitFor } from '@ember/test-helpers';
-import hbs from 'htmlbars-inline-precompile';
+import {
+  render,
+  settled,
+  waitFor,
+  type TestContext,
+} from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
 import { delay, test, visibilityClass } from '../../helpers/bootstrap';
 import setupNoDeprecations from '../../helpers/setup-no-deprecations';
```

Comment:
Converting `setupNoDeprecations` test helper to TypeScript is the only task pending for this PR as far as I'm aware. @simonihmig: Are you already working on that for #2093?